server/app.py

```diff
@@ -58,16 +61,32 @@ def get_parsed_qr(qr_type):
 
 @app.route('/recent-telemetry', methods=['GET'])
 def get_recent_telemetry():
-    response = jsonify(data=telem[-1],
```

Comment:
Can we follow the same return format as other endpoints? Don't think we need jsonify

Reply:
Will fix in future branch